docs/dao-setup.md

```diff
@@ -0,0 +1,83 @@
+# Bisq DAO setup guide
+
+## Configure Bitcoin Core
+
+If you want to run Bisq with DAO mode enabled you need to configure the `bitcoin.conf` file inside the Bitcoin Core data directory as well adding the `blocknotify` file.
+
+bitcoin.conf:
+
+    regtest=1
+
+    # The default rpcPort for regtest from Bitcoin Core 0.16 and higher is: 18443
+    # The default rpcPort for testnet is: 18332
+    # For mainnet: 8332
+    rpcport=18443
+
+    server=1
+    txindex=1
+    rpcuser=YOUR_USER_NAME
+    rpcpassword=YOUR_PW
+    blocknotify=bash [PATH TO DATA DIR]/blocknotify %s
+
+_Please note that `txindex` triggers a resync of the entire blockchain (be aware if you set that on mainnet as that it will take a while). Also take care if you use that setting for mainnet. Extra settings for more security are recommended in mainnet mode._
+
+The `blocknotify` file need to be added to the Bitcoin Core data directory as well:
+
+    #!/bin/bash
+    echo $1 | nc -w 1 127.0.0.1 5120
+    echo $1 | nc -w 1 127.0.0.1 5121
+    echo $1 | nc -w 1 127.0.0.1 5122
+    echo $1 | nc -w 1 127.0.0.1 5123
+
+It defines the ports where a new block event gets forwarded. Bisq will listen on that port and each Bisq node need to use a different port. You can add or remove ports from the list inside the file if needed.
+
+
+## Program arguments for DAO mode
```

Comment:
The arguments are mashed together when the markdown is rendered:
![image](https://user-images.githubusercontent.com/603793/48517112-2279c700-e81a-11e8-9d6a-cc0e3ca0be75.png)

Perhaps should make them into list items.